packages/crypto/src/coders/hex.spec.ts

```diff
@@ -3,10 +3,24 @@ import { hex } from "./hex";
 
 const ui8 = (bytes: number[]) => Uint8Array.from(bytes);
 
+const bytesTestVector = {
```

Comment:
For the `crypto` package in particular it might be worth looking into fuzz or prop testing in the future

We could probably test this with [`fuzz-check`](https://github.com/dubzzz/fast-check)

Reply:
Awesome suggestion! Already added, didn't know about `fast-check`. Thanks for your review.